seacliff/__init__.py

```diff
@@ -0,0 +1,12 @@
+try:
+    from importlib.metadata import version, PackageNotFoundError
+except ImportError:
+    from importlib_metadata import version, PackageNotFoundError
```

Comment:
This is confusing.  Why the try block?

Reply:
This is for cross version compat for python below 3.8. 

See https://github.com/python/importlib_metadata

Reply:
You may have missed the underscore versus dot. This package was only recently added to the stdlib.

Reply:
Ah! Yes, I thought this was the same thing twice.  lol